style.css

```diff
@@ -0,0 +1,386 @@
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.square {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.button2 {
+  display: flex;
+  justify-content: center;
+  padding: 12px;
+}
+
+.section6 {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  padding-left: 42px;
+  padding-right: 24px;
+}
+
+.quote {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  flex-direction: column;
+  text-align: center;
+}
+
+.holder-text {
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  border-radius: 12px;
+  margin: 0 25px 1px 25px;
+}
+
+.holder-text1 {
+  width: 100%;
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 24px;
+}
+
+.Language {
+  display: flex;
+  justify-content: center;
+  color: #344563;
+  font-size: 24px;
+  margin-top: -28px;
+  margin-bottom: 8px;
+  font-weight: 600;
+  line-height: 32px;
+  font-style: normal;
+  font-family: 'Courier New', Courier, monospace;
+}
+
+.media {
+  display: flex;
+  gap: 8px;
+  justify-content: center;
+  list-style-type: none;
+  color: #36b37f;
+  flex-wrap: wrap;
+  padding-right: 28px;
+  padding-left: 2px;
+}
+
+ul {
+  list-style: none;
+}
+
+.text-container ul {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 13px;
+  gap: 0.5rem;
+  padding: 20px 0;
+}
+
+.media li {
+  font-weight: 600;
+  background: white;
+  padding: 8px;
+  border-radius: 8px;
+  font-size: 12px;
+  line-height: 16px;
+  font-family: 'Courier New', Courier, monospace;
+}
+
+.text-container ul li {
+  color: #3a4a42;
+  background: #ebf0ee;
+  border-radius: 4px;
+  padding: 5px;
+}
+
+.holder-text1 img {
+  width: 100px;
+  height: 100px;
+}
+
+.about-p {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  margin: 20px 15px;
+  color: #172b4d;
+}
+
+.categories {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+  color: #3a4a42;
+  font-size: larger;
+}
+
+.image-container {
+  display: flex;
+  justify-content: center;
+  max-height: 500px;
+}
+
+.article {
+  display: flex;
+  justify-content: center;
+  gap: 2rem;
+  text-align: start;
+}
+
+.placeholder-box {
+  display: flex;
+  flex-direction: column;
+}
+
+.placeholder {
+  max-height: 250px;
+  width: 100%;
+  object-fit: cover;
+  object-position: center;
+  border-radius: 12px 12px 0 0;
+}
+
+.button-text:hover {
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+  transition-duration: 1s;
+}
+
+.button-text:focus {
+  background: #008552;
+}
+
+.text-about {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  grid: 1.5rem;
+}
```

Comment:
- [ ] I noticed the background image in the "About me" section is misaligned when compared with the Figma design and this leads to a bad UI so please make sure to fix this.
- Your design:
  ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/104968236/46be8531-d883-401a-982e-c43a041b641e)
- Figma design:
  ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/104968236/9e11ecf9-6ee0-4428-acd4-4655d5727377)